match-media.js

```diff
@@ -135,3 +148,60 @@ angular.module('matchMedia', [])
     return that.is(list);
   };
 }]);
+
+// Added my Matthias Max
```

Comment:
I'd prefer if you add a credit in the [contributors section of the readme](https://github.com/jacopotarantino/angular-match-media/#contributors) so that the code itself stays consistent.